engine/src/main/java/org/hibernate/validator/internal/cfg/context/ConfiguredConstraint.java

```diff
@@ -94,31 +97,24 @@ public ConstraintLocation getLocation() {
 	}
 
 	public ConstraintAnnotationDescriptor<A> createAnnotationDescriptor() {
-		return constraint.createAnnotationDescriptor();
+		return run( CreateConstraintAnnotationDescriptor.action( constraint ) );
```

Comment:
After giving it some thoughts and considering we only manipulate a private HV method, I'm wondering if we really need the SM here (I know I recommended to use it but I'm wondering now). The method can for sure be called on an external class but it's still the same `AnnotationDef`private method.

@gunnarmorling thoughts on this?